internal/xds/translator/cluster.go

```diff
@@ -301,6 +298,12 @@ func buildXdsClusterCircuitBreaker(circuitBreaker *ir.CircuitBreaker) *clusterv3
 				Value: *circuitBreaker.MaxParallelRequests,
 			}
 		}
+
+		if circuitBreaker.MaxParallelRetries != nil {
+			cbt.MaxRetries = &wrapperspb.UInt32Value{
+				Value: *circuitBreaker.MaxParallelRetries,
+			}
```

Comment:
lets be explicit with `1024` in case `circuitBreaker.MaxParallelRetries == nil`, in case we decide to run in non k8s env that doesnt set that default

Reply:
maybe we even need to set it if `circuitBreaker == nil`

Reply:
Sure, will do. For the other settings, I think that there's no reason to do it, as they anyway default to 1024 on envoy level.